.github/workflows/hadolint.yml

```diff
@@ -0,0 +1,14 @@
+name: Hadolint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    env:
+      HADOLINT_RECURSIVE: "true"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: hadolint/hadolint-action@v3.1.0
+        with:
+          dockerfile: "Dockerfile"
```

Comment:
Perhaps we should add test for all our dockerfiles? Maybe `dockerfile: "Dockerfile*"` could work? Or 
```code
dockerfile:
  - Dockerfile
  - Dockerfile.test
```